src/commands/install.ts

```diff
@@ -560,7 +556,7 @@ export async function run({
 
   let newLockfile: ImportMap | null = null;
   if (webDependencies && Object.keys(webDependencies).length > 0) {
-    newLockfile = await resolveTargetsFromRemoteCDN(lockfile, pkgManifest, config).catch((err) => {
+    newLockfile = await resolveTargetsFromRemoteCDN(lockfile, config).catch((err) => {
```

Comment:
This seems like an odd removal?

Reply:
lol I had the same thought. Looks like we were able to remove the arg from the function entirely

Reply:
The `resolveTargetsFromRemoteCDN` itself had an unused `pkgManifest` param. That cascaded through to these function calls.